db/migrate/20241125080445_add_villa_type_to_villa_units.rb

```diff
@@ -0,0 +1,5 @@
+class AddVillaTypeToVillaUnits < ActiveRecord::Migration[8.0]
+  def change
+    add_column :villa_units, :villa_type, :integer
```

Comment:
can rename columns?

-  `drop_point_status` to `down_payment_status `
-  `drop_point_paid_date ` to `down_payment_paid_date `